### ISSUE TITLE
Some ideas

### ISSUE THREAD

**Issue by darrepac** [lomorage/homepage#29]
Hello 

Some ideas to improve:
- allow to define an automatic rescan: define a folder to watch and a frequency (daily / weekly)
- Plex is doing its rescan with a low priority (nice value at 15) so it allows to not block the server's other tasks

my 2 cents :)

**Comment by fuji246** [comment]
Thanks for bring the ideas, just wondering what is the use scenario, for smart phone, we now have APP that can backup directly, and if using camera, those files will be on SD card and can import from there directly, either using the GUI import tool or command line tool, or using web browser.

**Comment by fuji246** [comment]
Or you just still want to manage those files by yourself, and keep the customized directory structure and just want to backup and be able to view those on smart phone/pad etc?

**Comment by darrepac** [comment]
> Or you just still want to manage those files by yourself, and keep the customized directory structure and just want to backup and be able to view those on smart phone/pad etc?

Yes that's it! and quite some users moving away from google photo or similar, are looking for this scenario.
